lib/web_ui/lib/src/engine/canvaskit/renderer.dart

```diff
@@ -0,0 +1,427 @@
+    positions,
+    textureCoordinates: textureCoordinates,
+    colors: colors,
+    indices: indices);
+
+  @override
+  ui.Vertices createVerticesRaw(
+    ui.VertexMode mode,
+    Float32List positions, {
+    Float32List? textureCoordinates,
+    Int32List? colors,
+    Uint16List? indices,
+  }) => CkVertices.raw(
+    mode,
+    positions,
+    textureCoordinates: textureCoordinates,
+    colors: colors,
+    indices: indices);
+
+  @override
+  ui.Canvas createCanvas(ui.PictureRecorder recorder, [ui.Rect? cullRect]) =>
+    CanvasKitCanvas(recorder, cullRect);
+
+  @override
+  ui.Gradient createLinearGradient(
+    ui.Offset from,
+    ui.Offset to,
+    List<ui.Color> colors, [
+    List<double>? colorStops,
+    ui.TileMode tileMode = ui.TileMode.clamp,
+    Float32List? matrix4
+  ]) => CkGradientLinear(from, to, colors, colorStops, tileMode, matrix4);
+
+  @override
+  ui.Gradient createRadialGradient(
+    ui.Offset center,
+    double radius,
+    List<ui.Color> colors, [
+    List<double>? colorStops,
+    ui.TileMode tileMode = ui.TileMode.clamp,
+    Float32List? matrix4,
+    ui.Offset? focal,
+    double focalRadius = 0.0,
+  ]) => CkGradientRadial(center, radius, colors, colorStops, tileMode, matrix4);
+
+  @override
+  ui.Gradient createConicalGradient(
+    ui.Offset focal,
+    double focalRadius,
+    ui.Offset center,
+    double radius,
+    List<ui.Color> colors,
+    [List<double>? colorStops,
+    ui.TileMode tileMode = ui.TileMode.clamp,
+    Float32List? matrix]
+  ) => CkGradientConical(
+    focal,
+    focalRadius,
+    center,
+    radius,
+    colors,
+    colorStops,
+    tileMode,
+    matrix);
+
+  @override
+  ui.Gradient createSweepGradient(
+    ui.Offset center,
+    List<ui.Color> colors, [
+    List<double>? colorStops,
+    ui.TileMode tileMode = ui.TileMode.clamp,
+    double startAngle = 0.0,
+    double endAngle = math.pi * 2,
+    Float32List? matrix4
+  ]) => CkGradientSweep(center, colors, colorStops, tileMode, startAngle, endAngle, matrix4);
+
+  @override
+  ui.PictureRecorder createPictureRecorder() => CkPictureRecorder();
+
+  @override
+  ui.SceneBuilder createSceneBuilder() => LayerSceneBuilder();
+
+  @override
+  ui.ImageFilter createBlurImageFilter({
+    double sigmaX = 0.0,
+    double sigmaY = 0.0,
+    ui.TileMode tileMode = ui.TileMode.clamp
+  }) => CkImageFilter.blur(sigmaX: sigmaX, sigmaY: sigmaY, tileMode: tileMode);
+
+  @override
+  ui.ImageFilter createDilateImageFilter({double radiusX = 0.0, double radiusY = 0.0}) {
+    // TODO(fzyzcjy): implement dilate. https://github.com/flutter/flutter/issues/101085
+    throw UnimplementedError('ImageFilter.dilate not implemented for CanvasKit.');
+  }
+
+  @override
+  ui.ImageFilter createErodeImageFilter({double radiusX = 0.0, double radiusY = 0.0}) {
+    // TODO(fzyzcjy): implement erode. https://github.com/flutter/flutter/issues/101085
+    throw UnimplementedError('ImageFilter.erode not implemented for web platform.');
```

Comment:
Should this also read "for CanvasKit" like `createDilateImageFilter` does? Or vice versa?